code/modules/cargo/bounties/assistant.dm

```diff
@@ -152,14 +147,14 @@
 	required_count = 8
 	wanted_types = list(/obj/item/twohanded/required/kirbyplants)
 
-/datum/bounty/item/assistant/earmuffs
-	name = "Earmuffs"
-	description = "Central Command is getting tired of your station's messages. They've ordered that you ship some earmuffs to lessen the annoyance."
-	reward = 1000
-	wanted_types = list(/obj/item/clothing/ears/earmuffs)
+// /datum/bounty/item/assistant/earmuffs
+// 	name = "Earmuffs"
+// 	description = "Central Command is getting tired of your station's messages. They've ordered that you ship some earmuffs to lessen the annoyance."
+// 	reward = 1000
+// 	wanted_types = list(/obj/item/clothing/ears/earmuffs)
 
-/datum/bounty/item/assistant/handcuffs
-	name = "Handcuffs"
+/datum/bounty/item/assistant/cuffs
+	name = "Cuffs"
```

Comment:
Undo this change, I don't see a reason an abbreviation should be used here.

Reply:
(As far as I know) The bounty can be completed by giving any kind of cuffs, not just handcuffs specifically. When giving out the bounty sheet, it will show as "Handcuffs", making people assume that only security handcuffs will fulfill it because they do not have access to the bounty description as cargo does.

Reply:
Yes, I know that, but they are still, by definition, handcuffs.